node-src/lib/getDependentStoryFiles.ts

```diff
@@ -80,7 +80,7 @@ export async function getDependentStoryFiles(
   // Convert a "webpack path" (relative to storybookBaseDir) to a "git path" (relative to repository root)
   // e.g. `./src/file.js` => `path/to/storybook/src/file.js`
   const normalize = (posixPath: FilePath): NormalizedName => {
-    const CSF_REGEX = /\s+sync\s+/g;
+    const CSF_REGEX = /\s+(sync|lazy)\s+/g;
```

Comment:
Technically, this is not needed, but since Weboack 5 and Rspack both have lazy paths, should this ensure those are ignored?

Reply:
Yeah I think that makes sense to me!